### PR TITLE
Friendly Documentation, for the better.

### PR DESCRIPTION
**Pull request by i0bs**
## About this pull request

This is new documentation for the Getting Started and Quickstart pages. This has been in the works for a long time and I have just now gotten around to publicizing it. The goal of this PR is to make it overall more user-friendly, and go into a more in-depth explanation of how things work with this library.

## Changes

Only .RST file changes to the docs.

## Checklist

- [ ] I've checked this pull request runs on `Python 3.6.X`.
- [ ] This fixes something in [Issues](https://github.com/eunwoo1104/discord-py-slash-command/issues).
    - Issue:
- [X] This adds something new.
- [ ] There is/are breaking change(s).
- [X] (If required) Relavent documentation has been updated/added.
- [X] This is not a code change. (README, docs, etc.)

### CODE REVIEW

**Comment by i0bs** [comment]
**FUTURE**: Plans to add documentation on how to use `try`/`catch` error handling on slash commands are considered right now, but I am waiting on the new hotfix Discord are planning to push to fix the `INTERACTION CREATE` errors for the API endpoint, as well as the new features.

File: docs/gettingstarted.rst [hunk 1]
```diff
@@ -1,93 +1,242 @@
...
+
+.. code-block:: python
+
+  from discord_slash.manage_commands import create_option
+  
+  @slash.slash(name="test",
+               description="This is just a test command, nothing more.",
+               options=[
+                 create_option(
+                   name="OptOne",
```
Comment (eunwoo1104) [inline docs/gettingstarted.rst]:
```suggestion
                   name="optone",
```
Same reason as above.

File: docs/gettingstarted.rst [hunk 2]
```diff
@@ -1,93 +1,242 @@
...
+                   description="This is the first option we have.",
+                   option_type=3,
+                   required=False
+                 )
+               ])
+  async def test(ctx, OptOne: str):
```
Comment (eunwoo1104) [inline docs/gettingstarted.rst]:
```suggestion
  async def test(ctx, optone: str):
```

File: docs/gettingstarted.rst [hunk 3]
```diff
@@ -1,93 +1,242 @@
...
+    await ctx.respond()
+    await ctx.send(content=f"I got you, you said {OptOne}!")
```
Comment (eunwoo1104) [inline docs/gettingstarted.rst]:
```suggestion
    await ctx.send(content=f"I got you, you said {optone}!")
```

**Review by eunwoo1104 — CHANGES_REQUESTED** [review]

**Review by eunwoo1104 — APPROVED** [review]

**Comment by eunwoo1104** [comment]
Merging, thank you for contributing!

**Pull request merged**